server/src/main/java/io/deephaven/server/hierarchicaltable/HierarchicalTableServiceGrpcImpl.java

```diff
@@ -332,6 +353,22 @@ private static Collection<SortColumn> translateAndValidateSorts(
         return translatedSorts;
     }
 
+    @Nullable
+    private static Collection<io.deephaven.api.Selectable> translateAndValidateUpdateView(
+            @NotNull final HierarchicalTableApplyRequest request,
+            @NotNull final BaseGridAttributes<?, ?> inputHierarchicalTable) {
+        if (request.getUpdateViewFormulasCount() == 0) {
+            return null;
+        }
+        final Collection<io.deephaven.api.Selectable> translated = request.getUpdateViewFormulasList().stream()
+                .map(AggregationAdapter::adapt)
+                .collect(Collectors.toList());
+
+        // TODO: what verification is needed here? We check illegal aliasing in the node recorder
```

Comment:
I would not think we would need to do anything about the aliasing here; but we need to look at formula application rules/allow lists for updateView over gRPC and make sure we are not bypassing them.